data/themes/darktable.css

```diff
@@ -648,7 +666,7 @@ combobox button
   border: 1px solid transparent;
   padding: 1px;
   background-color: transparent;
-  border-radius: 2pt;
+  border-radius: 2px;
```

Comment:
I don't think we want PX here. As far as I know PT is better as it translate properly on HiDPI screen.

Reply:
pt is normally not recommanded on screens. As css rules are maintain by w3c, see here : https://www.w3.org/Style/Examples/007/units.en.html
But as I don't use HiDPI screen (only full HD one), maybe GTK doesn't respect w3c rules. Could you confirm that ?

Reply:
I just read some things on gtk manual. The CSS properties page related to w3c pages. And I also read that HiDPI was added to GTK with using scale factor for HiDPI so I suppose that would be good.